src/Application.php

```diff
@@ -141,7 +146,7 @@ protected function initSabreDAV() {
                 $this['config']['auth']['type'],
                 $this['config']['auth']['realm'],
                 $this['pdo'],
-                null
+                '/dav.php'
```

Comment:
Ok.
But for me the browser plugin doesn't work when the base uri isn't set this way...
Any hints?